.github/workflows/claude-nl-suite.yml

```
@@ -102,80 +160,79 @@ jobs:
+              exited*|dead*)
+                docker logs unity-mcp --tail 200 | sed -E 's/((serial|license|password|token)[^[:space:]]*)/[REDACTED]/ig'
+                exit 1;;
+            esac
+            while :; do
+              l="$(docker logs unity-mcp 2>&1 || true)"
+              if echo "$l" | grep -qiE "$ok_pat"; then exit 0; fi
+              # Alternate readiness: parse bridge status JSON for unity_port and verify TCP connect on host
+              port="$(docker exec unity-mcp bash -lc 'python3 - <<PY\nimport glob,json,sys\nfor p in glob.glob("/root/.unity-mcp/unity-mcp-status-*.json"):\n  try:\n    with open(p,"r",encoding="utf-8") as f:\n      d=json.load(f)\n    if isinstance(d.get("unity_port"), int) and not d.get("reloading") and str(d.get("reason","")) in ("ready","idle"):\n      print(d["unity_port"])\n      sys.exit(0)\n  except Exception:\n    pass\nsys.exit(1)\nPY' 2>/dev/null || true)"
+              if [[ -n "$port" ]]; then
+                python3 - <<PY || true
+import socket,sys\ntry:\n  s=socket.socket()\n  s.settimeout(0.5)\n  s.connect(("127.0.0.1", int("$port")))\n  s.close()\n  sys.exit(0)\nexcept Exception:\n  sys.exit(1)
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 213: could not find expected ':'

(syntax-check)